src/Orchard.Cms.Web/Modules/Orchard.OpenId/JwtBearerConfigureOptions.cs

```diff
@@ -1,41 +0,0 @@
-﻿using Microsoft.AspNetCore.Builder;
```

Comment:
We do. And even if we wanted to make the options available to the services, we wouldn't needed an `IConfigureOptions<>`... an inline `ConfigureOptions` would be enough.